prometheus.yml

```diff
@@ -5,3 +5,16 @@
     - name: Printing Prometheus information
       debug:
         msg: "Prometheus would be installed and configured here."
+
+    - name: Prepare target | copy prometheus files
+      copy: src={{ item.src }} dest={{ item.dest }}
+      mode: 755
+      with_items:
+        - { src: 'redeploy-prometheus.sh', dest: '~/' }
+        - { src: 'prometheus.yml', dest: '~/' }
+        - { src: 'prometheus-high-performance.yaml', dest: '~/' }
+
+    # TODO:// grab prometheus-high-performance.yaml directly from github.
+
+    - name: deploy promethues
+      shell: sh ~/prometheus-high-performance.yaml deploy
```

Comment:
I am new to Ansible, but I have read that you use the `command` module over the `shell` module unless you need to do some pipe (|) or output redirection (>).

Reply:
ok, make sense we can replace it no problem.